client/src/components/v3/StatisticRow.tsx

```diff
@@ -24,14 +26,28 @@ export default function StatisticRow(): JSX.Element {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currVel]);
 
-  // TODO: Trap locations
+  // Previous Trap Speed (Achieved)
   // eslint-disable-next-line no-unused-vars
```

Comment:
You can remove this warning suppression (and the one below) since we're now using the variables!